README.md

```diff
@@ -1,3 +1,32 @@
-# fs-uploader
-[![Build Status](https://travis-ci.org/api-collab/fs-uploader.png)](https://travis-ci.org/api-collab/fs-uploader)
-[![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=io.apicollab%3Afs-uploader&metric=alert_status)](https://sonarcloud.io/dashboard?id=io.apicollab%3Afs-uploader)
\ No newline at end of file
+# api-collap-fs-uploader
+[![Build Status](https://travis-ci.org/api-collab/api-collab-fs-uploader.png)](https://travis-ci.org/api-collab/api-collab-fs-uploader)
+[![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=io.apicollab%3Afs%3Auploader&metric=alert_status)](https://sonarcloud.io/dashboard?id=io.apicollab%3Afs%3Auploader)
+
+This module makes it easy to upload new or updated api swagger spec to api portal.
+
+# Prerequisites
+- Java 8
+- Maven
+- Git
+
+# Running tests locally
+`
+mvn clean package
+` 
+# Deploy
+`
+java -jar target\fs-uploader.jar
+`
+# Configuration
+## File upload directory
+`${java.io.tmpdir}/api_input` folder is scanned for changes by default. This can be changed with `upload.baseDir` command line argument.
+## Api portal base url
```

Comment:
Is it worth mentioning the required vendor extensions in the swagger file?
x-application-name and any other ones